src/server-code/main-server.js

```diff
@@ -188,9 +190,6 @@ export default function runMainServer(serverFalcorModel) {
         }
       );
     });
-    app.get(/(?!\/login)/, (req, res) => {
-      res.redirect(307, `/login?url=${req.url}`);
-    });
   } else {
```

Comment:
When you remove the `app.get` block above you're gonna have to remove this else statement